app/utils/track.js

```diff
@@ -120,9 +120,12 @@ function isPageView(category) {
  * @returns {void}
  */
 export function track(params) {
+	const isFandomApp = new URL(window.location).searchParams.get('mobile-app') === 'true';
```

Comment:
this function runs also on Fastboot
on  line 133 and 137 there are neede checks - this should probably happen after these lines